packages/@lwc/errors/src/__tests__/errors.spec.ts

```diff
@@ -22,6 +22,8 @@ interface CustomMatchers<R = unknown> {
     toBeInRange: (range: { min: number; max: number }, key: string) => R;
 }
 
+import 'vitest';
```

Comment:
This is actually needed for the augmentation to work...

Reply:
Let's move this up to the top of the file. I'm surprised this doesn't fail the linter.